Shoko.Server/API/v3/Controllers/TmdbController.cs

```diff
@@ -1478,6 +1478,18 @@ public async Task<ActionResult> DownloadImagesForTmdbShowByShowID(
         return NoContent();
     }
 
+    /// <summary>
+    /// Download any missing TMDB Persons.
+    /// </summary>
+    /// <param name="removeErrors">Removes all references to the bad TMDB Person if unable to successfully download the person</param>
+    /// <returns> <see cref="OkResult"/> with a summary of found/updated/skipped/deleted actions. </returns>
+    [HttpGet("Person/DownloadMissing")]
+    public async Task<ActionResult> RepairMissingTmdbPersons([FromQuery] bool removeErrors = false)
+    {
+        var result = await _tmdbMetadataService.RepairMissingPersons();
```

Comment:
Since this a long running task then run it inside `Task.Run` and immediately return, and do the logging inside the metadata service instead.